src/app/workflow/workflowInfo.html

```diff
@@ -11,6 +11,9 @@
   Current state: <strong class="wd-active-state" ng-click="ctrl.selectAction(ctrl.workflow.state)">{{ctrl.workflow.state}}</strong>
   <span ng-show="ctrl.workflow.stateText">({{ctrl.workflow.stateText}})</span><br/>
   Current status: <strong>{{ctrl.workflow.status}}</strong><br/>
+  <div ng-show="ctrl.workflow.signal">
+    Current signal: <strong title="See supported signals tab in workflow definition">{{ctrl.workflow.signal}}</strong><br/>
```

Comment:
Show signal description (if available) in title (or somewhere) instead of "see supported signals..." text